e2e/cypress/tests/integration/performance/team_switch_spec.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel
+import {measurePerformance} from './utils.js';
+
+describe('Channel switch performance test', () => {
+    let testUser;
+    let testTeam1;
+    let testTeam2;
+
+    beforeEach(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam1 = team;
+
+            // # Login as test user and go to town square
+            cy.apiLogin(testUser);
+            cy.visit(`/${testTeam1.name}/channels/town-square`);
+        });
+
+        cy.apiCreateTeam('team-b', 'Team B').then(({team}) => {
+            testTeam2 = team;
+        });
+    });
+
+    it('measures switching between two teams from LHS', () => {
+        // # Invoke window object
+        measurePerformance('teamLoad', 5000, () => {
```

Comment:
Do we plan to fine tune the upperlimit here, or we leave it like this for now, and fine tune it in future iterations? (Probably we should be passing this upperlimits by some kind of environment variable, but 0/5)

Reply:
the next steps would be to fine tune the CI variables and logs, we have a ticket for that